ansible_collections/arista/avd/molecule/eos_designs_negative_unit_tests/inventory/host_vars/failure-connected-endpoint-port-profile-does-not-exist.yml

```diff
@@ -0,0 +1,24 @@
+---
+type: l3leaf
+
+port_profiles:
+  - profile: PROFILE-1
+
+l3leaf:
+  nodes:
+    - name: failure-connected-endpoint-port-profile-does-not-exist
+      loopback_ipv4_pool: 192.168.42.0/24
+      vtep_loopback_ipv4_pool: 192.168.255.0/24
+      bgp_as: 65042
+      id: 1
+
+servers:
+  - name: TEST-ENDPOINT
+    adapters:
+      - switches: [failure-connected-endpoint-port-profile-does-not-exist]
+        switch_ports: [Ethernet9]
+        profile: THIS-PROFILE-DOES-NOT-EXIST
+
+expected_error_message: >-
+  Profile 'THIS-PROFILE-DOES-NOT-EXIST' applied under 'servers[TEST-ENDPOINT].adapters'
```

Comment:
It would be nice to know which adapter. Maybe `servers[name=TEST-ENDPOINT].adapters[2]`

Reply:
please don't merge yet I am addressing this

Reply:
Fixed in [ae12665](https://github.com/aristanetworks/avd/pull/4516/commits/ae12665270f8ca23672d628566904db19d323563)